index.js

```diff
@@ -138,6 +138,7 @@ function bootstrap(options) {
 
   app.use(userMiddleware);
   app.use(hofMiddleware.cookies());
+  app.use(require('hof-middleware-markdown')());
```

Comment:
Fair.

I looked at the firearms implementation that's why I did it like that.  
https://github.com/UKHomeOffice/firearms/blob/master/app.js#L33

I wasn't sure what was the best approach.  I guess require above keeps it consistent and maybe is the best practice